doc/01-usage.md

```diff
@@ -149,6 +149,103 @@ error: options validation failed for image type "iot-raw-xz": ostree.url: requir
 $
 ```
 
+### bootc
+
+`image-builder` supports building images from [bootable containers](https://docs.fedoraproject.org/en-US/bootc/getting-started/). Building bootc-based images works differently from `ostree`-based images and package-based images.
+
+When building a bootable container into an image we try to base everything on the container. Thus the distribution that is being built is not known; you cannot use the `--distro` argument in combination with `--bootc-*` arguments as it would do nothing.
+
+The container(s) used for the various `--bootc-*` arguments must be in the container storage of the user running `image-builder` before the start of the build. This avoids needing to configure `image-builder` with appropriate credentials or access rights for container registries.
+
```

Comment:
This could use a mention of `--bootc-ref` before actually using it in the example. Something like "--bootc-ref is a reference to the container to be installed in the image"

Reply:
Makes sense, I've added a commit that explains it before the example and links to a page explaining what a container reference is.